deployments/compose/servers.yaml

```diff
@@ -16,12 +16,12 @@ services:
      - --debug
      - manufacturing
      - manufacturer:8038
-     - --manufacturing-key=/tmp/certs/manufacturer.key
-     - --owner-cert=/tmp/certs/owner.crt
-     - --device-ca-cert=/tmp/certs/device-ca.crt
-     - --device-ca-key=/tmp/certs/device-ca.key
+     - --manufacturing-key=/tmp/go-fdo/certs/manufacturer.key
```

Comment:
line 15 above seems to have a stray '\' char.  Not part of this patch but just noticed it.